core/src/modules/reset.ts

```diff
@@ -66,6 +66,8 @@ export namespace Reset {
       counts[model.name] = count;
     }
 
+    Property.invalidateCache();
```

Comment:
If we do more of these, I'd make some method like `System.invalidateCaches()`

Reply:
Yeah - for now I think `Property` is special. lets see!